apps/server/src/shared/domain/service/file-copy-append.service.ts

```diff
@@ -1,5 +1,10 @@
 import { Injectable } from '@nestjs/common';
-import { FileDto, FileParamBuilder, FilesStorageClientAdapterService } from '@src/modules/files-storage-client';
+import {
+	FileParamBuilder,
+	FilesStorageClientAdapterService,
+	CopyFilesService,
+} from '@src/modules/files-storage-client';
+import { CopyFileResponse } from '@src/modules/files-storage/controller/dto';
```

Comment:
This must be fixed. The append service should not use any stuff from seperated files-storage. 
They only import FilesStorageClientAdapterService as requirement and need his values from this place. 

It should not pull directly from open-api-client export because it need a internal representation of what they need and in future maybe it is not the same like the export from file-service.